[libmagic] build failure

### Operating system

x64-windows

### Compiler

MSVC

### Steps to reproduce the behavior

```Shell
vcpkg install libmagic
```

### Failure logs

Installing 92/120 libmagic:x64-windows@5.46...
Building libmagic:x64-windows@5.46...
Downloading https://github.com/file/file/archive/FILE5_46.tar.gz -> file-file-FILE5_46.tar.gz
D:\a\docwire\docwire\vcpkg\downloads\file-file-FILE5_46.tar.gz.740.part: error: download from https://github.com/file/file/archive/FILE5_46.tar.gz had an unexpected hash
note: Expected: 9165bb5bdbe7b8fccac0c8675d4eb251a286ab2ab7a79e6f8ed98d36fa0928b889cf109c1da3a5cfff64d1b1006b5d73934c2d420484adae6f4c8e26a9ede18f
note: Actual  : fb8157ee8065feaf57412ccdeee57cd8fc853b54ac49b0ddc818eeb1ca3555a7cfd25dea08996503f7c565dcba8c57fd7e4dc5fe3452872c617f5612a94a8f0e
CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:124 (message):
  Download failed, halting portfile.
Call Stack (most recent call first):
  scripts/cmake/vcpkg_from_github.cmake:106 (vcpkg_download_distfile)
  ports/libmagic/portfile.cmake:21 (vcpkg_from_github)
  scripts/ports.cmake:196 (include)

See also: https://github.com/docwire/docwire/actions/runs/13065608685/job/36491110203

### Additional context

_No response_

--- a/ports/libmagic/portfile.cmake
+++ b/ports/libmagic/portfile.cmake
@@ -22,7 +22,7 @@ vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO file/file
     REF FILE5_46
-    SHA512 9165bb5bdbe7b8fccac0c8675d4eb251a286ab2ab7a79e6f8ed98d36fa0928b889cf109c1da3a5cfff64d1b1006b5d73934c2d420484adae6f4c8e26a9ede18f
+    SHA512 fb8157ee8065feaf57412ccdeee57cd8fc853b54ac49b0ddc818eeb1ca3555a7cfd25dea08996503f7c565dcba8c57fd7e4dc5fe3452872c617f5612a94a8f0e
     HEAD_REF master
     PATCHES ${PATCHES}
 )

--- a/ports/libmagic/vcpkg.json
+++ b/ports/libmagic/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "libmagic",
   "version": "5.46",
+  "port-version": 1,
   "description": "This library can be used to classify files according to magic number tests.",
   "homepage": "https://github.com/file/file",
   "license": "BSD-2-Clause",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -4774,7 +4774,7 @@
     },
     "libmagic": {
       "baseline": "5.46",
-      "port-version": 0
+      "port-version": 1
     },
     "libmariadb": {
       "baseline": "3.4.1",

--- a/versions/l-/libmagic.json
+++ b/versions/l-/libmagic.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "64418c4e61811c2e8829f8253c56173a2836bbcd",
+      "version": "5.46",
+      "port-version": 1
+    },
+    {
       "git-tree": "380e305405e20a74576fcee51673a98a5a49fe18",
       "version": "5.46",
       "port-version": 0

